r-rpostgresql-feedstock/recipe/meta.yaml

```diff
@@ -42,13 +42,13 @@ requirements:
   host:
     - r-base
     - r-dbi >=0.3
-    - postgresql
+    - postgresql 17.0
 
   run:
     - r-base
     - {{native}}gcc-libs         # [win]
     - r-dbi >=0.3
-    - postgresql
+    - {{ pin_compatible('postgresql', max_pin='x.x.x') }}
```

Comment:
Is there a reason for such a strict pinning?
https://github.com/AnacondaRecipes/postgresql-feedstock/blob/master/recipe/meta.yaml#L58 here is a run_export which will add `libpq >=17.0`. Leaving simply `- postgresql` should be sufficient.
(If we are too strict on our pinning, that means we will have to rebuild this package even if we update from 17.0 to 17.1.)

Reply:
Turns out, reading is hard. My intention was to do `min_pin='x.x.x'` rather than `max_pin`.

Assuming I had committed the intended bounds, I see how it would end up as basically the same thing as what the `run_export` evaluates to.